src/Asana/Errors/AsanaError.php

```diff
@@ -23,6 +24,16 @@ public static function handleErrorResponse($response)
     {
         switch ($response->code) {
             case ForbiddenError::STATUS:
+                // Handle 403's with a premium response
+                try {
+                    foreach ($response->body->errors as $error) {
+                        if (strpos($error->message, self::PREMIUM_ONLY_STR) !== false) {
+                            throw new PremiumOnlyError($response);
+                        }
+                    }
+                } catch (Exception $e) {
```

Comment:
I wouldn't have expected this to need a `try`/`catch`, but it does no harm and is temporary anyway.

Reply:
Yeah, I'm just keeping it consistent with the other libraries; agree on that.